Block/Form/Cc.php

```diff
@@ -70,27 +64,35 @@ public function __construct(
     )
     {
         parent::__construct($context, $paymentConfig);
-        $this->_adyenHelper = $adyenHelper;
-        $this->_appState = $context->getAppState();
-        $this->_checkoutSession = $checkoutSession;
+        $this->adyenHelper = $adyenHelper;
+        $this->appState = $context->getAppState();
+        $this->checkoutSession = $checkoutSession;
     }
 
+	/**
+	 * @return string
+	 */
+    public function getCheckoutCardComponentJs()
+	{
+		return $this->adyenHelper->getCheckoutCardComponentJs($this->checkoutSession->getQuote()->getStore()->getId());
+	}
 
-    /**
-     * @return mixed
-     * @throws \Magento\Framework\Exception\LocalizedException
-     */
-    public function getLibrarySource()
-    {
-        // get storeId for admin
-        if (!$this->_appState->getAreaCode() === \Magento\Backend\App\Area\FrontNameResolver::AREA_CODE) {
-            $storeId = $this->_storeManager->getStore()->getId();
-        } else {
-            $storeId = null;
-        }
+	/**
+	 * @return string
+	 * @throws \Adyen\AdyenException
+	 */
+	public function getCheckoutOriginKeys()
+	{
+		return $this->adyenHelper->getOriginKeyForBaseUrl();
```

Comment:
please provide storeId as storeId can be different foreach checkout as this is related to the store you have selected.